scripts/globals/mobs.lua

```diff
@@ -59,14 +59,16 @@ function phOnDespawn(ph,phList,chance,cooldown)
         local nm = GetMobByID(nmId);
         if (nm ~= nil) then
             local pop = nm:getLocalVar("pop");
-            if (os.time() > pop and not lotteryPrimed(phList) and math.random(100) <= chance) then
+            local flag = nm:getLocalVar("flag");
```

Comment:
could we call that something more descriptive?

Reply:
trying hard ... :)